test/pointer/seman/Pointer_atFieldUnboxedBad20.v3

```diff
@@ -0,0 +1,13 @@
+//@seman=TypeError @ 11:17
+// tests possible issues related to unboxed field and parameterized types
+
+component Comp {
+	var f: void;
+}
+
+def main(n: int) -> int {
```

Comment:
Seman tests don't need to have a `main`, because they don't execute. We can simplify this down to just:
```
component C {
    var f: void;
    var p = Pointer.atField(f);
}
```

Reply:
In general, the fewer lines of code needed to get to a semantic error, the better. Helps a lot in debugging and makes tests run faster.